source/features/widgets/footer/components/Footer.scss

```diff
@@ -8,13 +8,18 @@
 }
 
 .footerContainer {
-  background-color: $primary-bg;
-  padding: 78px 40px;
+  width: 100%;
+
+  .footerContent {
+    width: 872px;
```

Comment:
Alphabetic ordering

Reply:
This is WIP pr. Need to update several components, will do this sorting once they are done.